### DISCUSSION TITLE
Saving Images from the images entity

### DISCUSSION THREAD

**Discussion by sheminasalam** [blakeblackshear/frigate#9725] [Ask A Question]
With the latest update, the camera entities which showed the latest detection has been replaced with images entities. Earlier I was using the snapshot service to save the latest detection image from the camera entity now I can't do that. Is there any way to save the latest image of the images entity to a specific folder in HA?

**Comment by NickM-27** [comment]
this is a question for home assistant more than frigate

**Comment by sheminasalam** [comment]
but this change was implemented by frigate. HA still supports camera entities.

**Reply by NickM-27** [reply]
you are asking about how to do something in Home Assistant, home assistant would be the place to answer that question
